_header

```diff
@@ -0,0 +1,10 @@
+Cache-Control: no-cache, no-store, max-age=0, must-revalidate
```

Comment:
Why do you want to configure no-cache headers?
We should always aim for the exact opposite.

Reply:
I disagree and so does OWASP: https://www.owasp.org/index.php/OWASP_Application_Security_FAQ#Browser_Cache. Cache should be set deliberately set on objects than should be cached. The default should be off.

Reply:
Of course. My response was too generic to be true. We should always aim for caching, but of course rather cache too few then too much/long.

As this is a static website I would go for 
`Cache-Control: public, max-age=3600`

I'm not aware of a more fine granulare way to control cache headers with netlify. If so we could of course go for this as a standard and then define more specific cache headers for each resource type (html, css, js, ...)

Reply:
Done.